prompt.py

```diff
@@ -933,6 +941,32 @@ def run(self):
                 traceback.print_exc()
 
 
+def check_privatenet():
+    """ Check if privatenet is running, and if container is same as the chain file """
+    host = "http://127.0.0.1:30333"
```

Comment:
Can we read this from the settings/protocol.privnet.json (or whatever option is used to indicate which network to use) instead? I don't think we should assume people only run a privnet on their local machine.

Reply:
done